src/lua_jailbreak_exception.cpp

```diff
@@ -52,7 +56,9 @@ void lua_jailbreak_exception::rethrow()
 	try {
 		jailbreak_exception->execute();
 	} catch(...) {
-		clear();
+		if (!jail_depth) {
```

Comment:
I think we'd prefer `jail_depth == 0` rather than `!jail_depth`.

Reply:
I would too, actually.  [Amended](https://github.com/wesnoth/wesnoth/compare/cf21c233244..d7e5bd4553b), thanks.